site/layouts/partials/footer.html

```diff
@@ -12,10 +12,10 @@ <h3 class="accordion-header footer-heading" id="headingLinks">
           </h3>
           <div id="collapseLinks" class="container-xxl accordion-collapse collapse" data-bs-parent="#sitemapAccordion">
             <ul class="navbar-nav">
-              <li><a class="nav-link" href="/" aria-describedby="headingLinks">Home</a></li>
-              <li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/" aria-describedby="headingLinks">Docs</a></li>
-              <li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/examples/" aria-describedby="headingLinks">Examples</a></li>
-              <li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/guidelines/" aria-describedby="headingLinks">Design guidelines</a></li>
+              <li><a class="nav-link{{ if .IsHome }} active" aria-current="page{{ end }}" href="/" aria-describedby="headingLinks">Home</a></li>
+              <li><a class="nav-link{{ if eq .Page.Layout "docs" }} active" aria-current="true{{ end }}" href="/docs/{{ .Site.Params.docs_version }}/" aria-describedby="headingLinks">Docs</a></li>
+              <li><a class="nav-link{{ if eq .Page.Title "Examples" }} active" aria-current="page{{ end }}" href="/docs/{{ .Site.Params.docs_version }}/examples/" aria-describedby="headingLinks">Examples</a></li>
+              <li><a class="nav-link{{ if eq .Page.Title "Orange Design System for web" }} active" aria-current="page{{ end }}" href="/docs/{{ .Site.Params.docs_version }}/guidelines/" aria-describedby="headingLinks">Design guidelines</a></li>
               <!-- Boosted mod: removed .Site.Params.icons|themes|blog|swag -->
             </ul>
           </div>
```

Comment:
Also wondering what to do with lines 32 to 35:

```
<li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/getting-started/" aria-describedby="headingGuides">Getting started</a></li>
<li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/getting-started/webpack/" aria-describedby="headingGuides">Webpack</a></li>
<li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/getting-started/parcel/" aria-describedby="headingGuides">Parcel</a></li>
<li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/getting-started/vite/" aria-describedby="headingGuides">Vite</a></li>
```

These are not links to pages but to multiples sections of the documentation so I am still wondering what's the best way to handle them?

Reply:
IMO (like Julien would say ;-) ), for "Getting started", it would be nice to have this link marked as active while the url starts with ".../docs/5.3/getting-started/" except for the subparts Parcel, Vite and Webpack
And these 3 links marked as active when you're on the corresponding page...